session/mock_bootstrap.go

```diff
@@ -39,8 +40,6 @@ var allDDLs = []string{
 	"alter table mock_sys_t add column c4 bigint",
 	"create table test_create_table(a int)",
 	"drop table test_create_table",
-	"create database test_create_db",
-	"drop database test_create_db",
```

Comment:
Why remove these?

Reply:
Because these aren't the system database, so if we put them in here, they're paused, so they can't execute normally.